### ISSUE TITLE
CTFE does not warn arithmetic overflow of literal integer references

### ISSUE THREAD

**Issue by c410-f3r** [rust-lang/rust#101283]
Feel free to close if duplicated.

```rust
fn main() {
    // Panics at runtime
    let _: u32 = &u32::MAX + &1;
}
```

**Comment by c410-f3r** [comment]
Closing due to the lack of activity.
